utils/gasInfo.go

```diff
@@ -0,0 +1,44 @@
+package utils
+
+import (
+	"encoding/json"
+	"log"
+	"math"
+	"math/big"
+	"net/http"
+)
+
+// SuggestedGasPrice returns the fast gas price that ethGasStation
+// recommends for transactions to be mined on Ethereum blockchain.
+func SuggestedGasPrice() *big.Int {
+	request, _ := http.NewRequest("GET",
+		"https://ethgasstation.info/json/ethgasAPI.json", nil)
+
+	request.Header.Set("Content-Type", "application/json")
+
+	client := &http.Client{}
+	response, err := client.Do(request)
+	if err != nil {
+		log.Printf("cannot connect to ethGasStationAPI: %v", err)
+		return nil
+	}
+
+	if response.StatusCode != http.StatusOK {
+		log.Printf("received status code %v from ethGasStation",
+			response.StatusCode)
```

Comment:
This kind of new-lining is inconsistent with style. Have you run gofmt?

Reply:
It should just be

```go
log.Printf("received status code %v from ethGasStation", response.StatusCode)
```

But, for very long lines you can use

```go
log.Printf(
    "received status code %v from ethGasStation",
    response.StatusCode,
)
```

Reply:
Yup, gofmt hadn't complained. I will change to the second formatting.